MediaBrowser.Controller/BaseItemManager/BaseItemManager.cs

```diff
@@ -12,15 +14,33 @@ public class BaseItemManager : IBaseItemManager
     {
         private readonly IServerConfigurationManager _serverConfigurationManager;
 
+        private int _metadataRefreshConcurrency = 0;
+
         /// <summary>
         /// Initializes a new instance of the <see cref="BaseItemManager"/> class.
         /// </summary>
         /// <param name="serverConfigurationManager">Instance of the <see cref="IServerConfigurationManager"/> interface.</param>
         public BaseItemManager(IServerConfigurationManager serverConfigurationManager)
         {
             _serverConfigurationManager = serverConfigurationManager;
+
+            _metadataRefreshConcurrency = GetMetadataRefreshConcurrency();
+            SetupMetadataThrottler();
+
+            _serverConfigurationManager.ConfigurationUpdated += (object sender, EventArgs e) =>
```

Comment:
Can you make this into a "real" function instead ie. `private void OnConfigurationUpdated(object sender, EventArgs e)`?